### PR TITLE
update learn service mesh section 

### PR DESCRIPTION
**Pull request by Tanuj22**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by Nikhil-Ladha — COMMENTED** [review]
When we go back, all the labs are not showing up. Only one lab is coming `Working with Meshery and NSM`

File: src/sections/Learn/LearnPage-Sections/learn.js [hunk 1]
```diff
@@ -166,33 +184,38 @@ const LearnSection = () => {
       </div>
       <div>
         <h1 className="learn-heading"><span>Meshery - </span> Learn how to manage your service mesh</h1>
+        <Container>
+          <Row Hcenter className="learn-buttons-section labs">
+            <Col xs={12} lg={6}>
+              <Link to="/learn/service-mesh-labs" state={{selectedIndex: 0}}>
+                <Button secondary title="Getting started with any service mesh"/>
```
Comment (Nikhil-Ladha) [inline src/sections/Learn/LearnPage-Sections/learn.js:191]: @leecalcote links for these buttons?

File: src/sections/Learn/LearnPage-Sections/learn.js [hunk 2]
```diff
@@ -166,33 +184,38 @@ const LearnSection = () => {
...
+              </Link>
+            </Col>
+            <Col xs={12} lg={6}>
+              <Link to="/learn/service-mesh-labs" state={{selectedIndex: 1}}>
+                <Button secondary title="Managing the performance of your service mesh"/>
```
Comment (Nikhil-Ladha) [inline src/sections/Learn/LearnPage-Sections/learn.js:196]: And this one too, @leecalcote

**Comment by Nikhil-Ladha** [comment]
> When we go back, all the labs are not showing up. Only one lab is coming `Working with Meshery and NSM`

One thing to note, this is happening only if we go from `/learn` to individual labs using the cards link. But, if we got directly to `/learn/service-mesh-labs` it's not happening.

File: src/sections/Learn/LearnPage-Sections/learn.js (hunk 1 above)
Comment (Tanuj22) [reply src/sections/Learn/LearnPage-Sections/learn.js:191]: Links are added in the `Link` to pass the prop

**Comment by Nikhil-Ladha** [comment]
Error from console:
```
index.js:2177 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in Navigation (created by IndexPage)
    in Layout (created by IndexPage)
    in ThemeProvider (created by IndexPage)
    in IndexPage (created by HotExportedIndexPage)
    in AppContainer (created by HotExportedIndexPage)
    in HotExportedIndexPage (created by PageRenderer)
```

File: src/sections/Learn/LearnPage-Sections/learn.js (hunk 1 above)
Comment (Nikhil-Ladha) [reply src/sections/Learn/LearnPage-Sections/learn.js:191]: I have seen that, but I feel it should be changed.

Comment (leecalcote) [reply src/sections/Learn/LearnPage-Sections/learn.js:191]: I see what you mean, @Nikhil-Ladha. You're suggesting that the two buttons use the same hyperlink and don't deep link to one of the two (green and yellow) tabs representing the two different categories of labs.

Yes, if we can deep link to have the respective tab active, that would be an improvement.

Comment (leecalcote) [reply src/sections/Learn/LearnPage-Sections/learn.js:191]: No doubt that this is what the `state={{selectedIndex: x }}` property is about in the Link now., which in my tests is working.

Comment (leecalcote) [reply src/sections/Learn/LearnPage-Sections/learn.js:191]: I take it back. I'm only seeing the NSM lab when clicking on the "getting started" button.

**Comment by Tanuj22** [comment]
> Error from console:
> 
> ```
> index.js:2177 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
>     in Navigation (created by IndexPage)
>     in Layout (created by IndexPage)
>     in ThemeProvider (created by IndexPage)
>     in IndexPage (created by HotExportedIndexPage)
>     in AppContainer (created by HotExportedIndexPage)
>     in HotExportedIndexPage (created by PageRenderer)
> ```

I am not seeing this :thinking:

**Comment by Nikhil-Ladha** [comment]
@leecalcote see, in this PR since it was pushed from a forked repo the `build and deploy` action was not executed.

**Comment by leecalcote** [comment]
> @leecalcote see, in this PR since it was pushed from a forked repo the `build and deploy` action was not executed.

Oh, ok. Are you thinking that adding `branch: master` might solve this? We can change it any which way (no opinion here)

**Review by leecalcote — CHANGES_REQUESTED** [review]
The links to the service mesh labs are opening katacoda.com in a new window.

**Comment by leecalcote** [comment]
Alright, fine. 

I'm a liar.

...and I had the wrong branch checked out. Sue me.

**Comment by leecalcote** [comment]
Screen Shot 2021-02-26 at 9 32 09 AM
 a little less vertical space between the logo and text might be nice.

**Review by leecalcote — APPROVED** [review]
Thank you @Tanuj22

**Pull request merged**